cmd/convox/env.go

```diff
@@ -282,3 +290,25 @@ func cmdEnvUnset(c *cli.Context) error {
 
 	return nil
 }
+
+func parseEnvLine(line string) (key string, value string, err error) {
+	// Deal with empty lines
+	var m bool
+	if m = regexp.MustCompile(`^\s*$`).MatchString(line); m {
+		return
```

Comment:
agreed - I liked named vars for the self documenting properties, you are right to it should match the style of the rest of the code.  I have added a 1 liner documenting description.